website/docs/r/rights_bundle.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "vcfa"
+page_title: "VMware Cloud Foundation Automation: vcfa_rights_bundle"
+sidebar_current: "docs-vcfa-resource-rights-bundle"
+description: |-
+ Provides a VMware Cloud Foundation Automation Rights Bundle resource. This can be used to create, modify, and delete Rights Bundles.
+---
+
+# vcfa\_rights\_bundle
+
+Provides a VMware Cloud Foundation Automation Rights Bundle resource. This can be used to create, modify, and delete Rights Bundles.
+
+## Example Usage
+
+```hcl
+resource "vcfa_rights_bundle" "new-rights-bundle" {
+  name        = "new-rights-bundle"
+  description = "new rights bundle from CLI"
```

Comment:
How to understand "**from** CLI"?

Reply:
Changed to `"new rights bundle from Terraform"`, we had this "CLI" in VCD docs, not sure what it means in that context, tbh. Looks like a typo to me.